app/sidekiq/evss/disability_compensation_form/form4142_document_upload_failure_email.rb

```diff
@@ -71,7 +77,7 @@ def perform(form526_submission_id)
           first_name = form526_submission.get_first_name
           date_submitted = form526_submission.format_creation_time_for_mailers
 
-          @notify_client.send_email(
+          notify_response = @notify_client.send_email(
```

Comment:
I didn't add it, that was what was already here. The only way it would be valuable if multiple methods in this same class needed to use this var and we only wanted it instantiated once. But since this is in a job block that is only going to be called once, I agree that doesnt need to be an instance var

Reply:
addressed in c5dd50aea5